.codeclimate.yml

```diff
@@ -0,0 +1,15 @@
+---
+engines:
+  duplication:
+    enabled: true
+    config:
+      languages:
+      - ruby
+  fixme:
+    enabled: true
+  rubocop:
+    enabled: false
+ratings:
+  paths:
+  - "**.rb"
```

Comment:
I think it make sense to use "src/**.rb" otherwise it will count also tests in test directory, which is not so useful